fineract-client/build.gradle

```diff
@@ -87,12 +87,23 @@ task buildTypescriptAngularSdk(type: org.openapitools.generator.gradle.plugin.ta
     ]
 }
 
-licenseFormatMain.dependsOn buildJavaSdk
```

Comment:
Have you tested the incremental build without these? i.e. executing integration tests from IntelliJ should not regenerate the fineract-client if no changes.

Reply:
I've put a working set of deps together now... to be honest: don't know who came up with those "licenseFormatMain" dependencies. It is "licenseMain" now and that works now. I've removed also those compile task deps... that's like doing things twice. Let's see what the build says.